bokeh/models/axes.py

```diff
@@ -26,7 +26,7 @@ class Axis(GuideRenderer):
     Where should labels and ticks be located in relation to the axis rule.
     """)
 
-    bounds = Either(Auto, Tuple(Float, Float), help="""
+    bounds = Either(Auto, Tuple(Float, Float), Tuple(Datetime, Datetime), help="""
```

Comment:
Do we know if this works for sure? I feel like I tried this and it was not sufficient but I may be misremembering.